src/Elm/Interface.elm

```diff
@@ -119,46 +121,42 @@ operators =
 build : RawFile -> Interface
 build (Raw file) =
     let
+        fileDefinitionList : List ( String, Exposed )
         fileDefinitionList =
             fileToDefinitions file
     in
     case Module.exposingList (Node.value file.moduleDefinition) of
         Explicit x ->
-            buildInterfaceFromExplicit x fileDefinitionList
+            buildInterfaceFromExplicit x (Dict.fromList fileDefinitionList)
 
         All _ ->
             List.map Tuple.second fileDefinitionList
 
 
-lookupForDefinition : String -> List ( String, Exposed ) -> Maybe Exposed
-lookupForDefinition key =
-    List.filter (Tuple.first >> (==) key) >> List.head >> Maybe.map Tuple.second
```

Comment:
This was basically a slow `Dict.get`. The calls to this function were therefore replaced by `Dict.get` and the data they were working on transformed to a `Dict`.